base/services/w32time/w32time.c

```diff
@@ -149,8 +149,8 @@ GetIntervalSetting(VOID)
         RegCloseKey(hKey);
     }
 
-    if (lRet != ERROR_SUCCESS)
-        return 0;
+    if (lRet != ERROR_SUCCESS || dwData < 120)
+        return 32400;
```

Comment:
`9 * 60 * 60` would be clearer. And since 9 hours is kind of an odd duration, perhaps a comment for the rationale would be useful?

Reply:
I used 9 hours because of @MailYouLater commented

> Also, I checked what this registry value is set to in an install of Windows 10, and it's 32768 which would be 9 hours, 6 minutes and 8 seconds.

I liked 9 because it's close to what MS uses :P

Reply:
I understand -- it's just that that's not apparent from the code or the commit message, and you generally shouldn't have to read PR comments to figure out the reasoning.

Reply:
I'll leave a comment :)